bgw-core/src/main/kotlin/tools/aqua/bgw/builder/UINodeBuilder.kt

```diff
@@ -297,6 +297,16 @@ internal class UINodeBuilder {
 			addListener { _, _, new -> labeled.text = new }
 		}
 
+		/**
+		 * Binds [TextInputUIComponent.promptProperty].
+		 */
+		private fun javafx.beans.property.StringProperty.bindPromptProperty(labeled: TextInputUIComponent) {
+			//Framework -> JavaFX
+			labeled.promptProperty.setGUIListenerAndInvoke(labeled.prompt) { _, nV -> value = nV }
+			//JavaFX -> Framework
+			addListener { _, _, new -> labeled.prompt = new }
```

Comment:
Can be omitted since there is no way a user could change the prompt from the GUI.